vector/src/main/AndroidManifest.xml

```diff
@@ -8,6 +8,7 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
     <uses-permission android:name="android.permission.VIBRATE" />
     <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
this is needed for all android versions?

Reply:
Yes, it is needed to read files from external storage and no it is not since we request WRITE_EXTERNAL_STORAGE which encapsulates read permission.